### ISSUE TITLE
Manager mode

### ISSUE THREAD

**Issue by shilman** [joshbolduc/vscode-story-explorer#1056]
Hi @joshbolduc, Storybook maintainer here. This is an amazing extension!!!!

Currently you are showing Storybook's "preview" aka `iframe.html` in the panel. I'd love a configuration option to show Storybook's "manager" (outer iframe) instead. I hacked it together with the following code in `entry-story.ts`, however I didn't figure out how to add this as a configuration option and get the data from `getConfiguration` into the iframe.

```ts
const previewUrl = (storyId: string, storyType: string) => {
  // For docs, we (sometimes) rely on storybook figuring out the full ID for us,
  // which requires the full story store to be loaded. (Investigate whether this
  // is also needed to support <Story /> elements that reference other stories,
  // in which case we really can't use `singleStory` for docs.)
  const useSingleStory = storyType === 'story';

  return `/iframe.html?id=${encodeURIComponent(
    storyId,
  )}&viewMode=${encodeURIComponent(storyType)}${
    useSingleStory ? '&singleStory=true' : ''
  }`;
};

const managerUrl = (storyId: string, storyType: string) => {
  return `/?path=/${encodeURIComponent(storyType)}/${encodeURIComponent(
    storyId,
  )}&nav=0&panel=bottom`;
};
```

Do you think this could be a useful contribution? If so, can you advise on the best path forward? I'd be happy to create a PR.

**Comment by joshbolduc** [comment]
I noticed a difference between the approach here and what was proposed in #1046. It looks like it comes down to the `singleStory` query parameter.

With (no hamburger menu in top toolbar):
<img width="1912" alt="image" src="https://github.com/user-attachments/assets/5089a5fe-d536-4614-bfab-31de473c659e" />

Without (hamburger menu visible):
<img width="1912" alt="image" src="https://github.com/user-attachments/assets/d805992f-8cb6-4bb7-87a6-653044e409f5" />

(It's also a more obvious difference running Storybook 7 instead of 8.)

They both look the same to me when the webview is narrow enough, and let you open the sidebar from the bottom (even in single story mode):
<img width="1912" alt="image" src="https://github.com/user-attachments/assets/ee9122e5-a570-49b1-9f19-606de7d53ab2" />

I think having the option to show toolbars is valuable and makes sense. Showing the sidebar navigation is a little more awkward given the way the extension works today, which assumes a 1:1 relationship between each story and the webview for that story. So by exposing the sidebar in the webview, you can navigate to a different story than the extension thinks is being shown in that webview, and things get out of sync.

Given that, it seems like including `singleStory=true` is the way to go, but I might be missing something?

**Comment by shilman** [comment]
@joshbolduc Good call on `singleStory=true`! 💯

**Comment by joshbolduc** [comment]
This has been released in [1.1.0](https://github.com/joshbolduc/vscode-story-explorer/releases/tag/v1.1.0)!
